tests/services/scanner/scanner_dao_test.py

```diff
@@ -60,21 +63,55 @@
      }
 ]
 
-def populate_db():
-    """Populate the db with violations."""
-    engine = create_test_engine()
-    violation_access_cls = scanner_dao.define_violation(engine)
-    violation_access = violation_access_cls(engine)
-    violation_access.create(FAKE_EXPECTED_VIOLATIONS,
-                            FAKE_INVENTORY_INDEX_ID)
-    return violation_access 
 
-class ScannerDaoTest(ForsetiTestCase):
-    """Test scanner data access."""
+class DatabaseTest(ForsetiTestCase):
+    """Base class for database centric tests."""
 
     def setUp(self):
         """Setup method."""
         ForsetiTestCase.setUp(self)
+        self.engine, self.dbfile = create_test_engine_with_file()
+        _session_maker = sessionmaker()
+        self.session = _session_maker(bind=self.engine)
+        storage.initialize(self.engine)
+        scanner_dao.initialize(self.engine)
+        self.session.flush()
+        self.violation_access = scanner_dao.ViolationAccess(self.session)
+        self.iidx_id1, self.iidx_id2, self.iidx_id3 = _setup_inv_indices(
```

Comment:
Change the `iidx` to `inv_index_id` to be consistent with the application code.  Please fix everywhere.

Reply:
Done.